aria/modeling/service_template.py

```diff
@@ -33,7 +33,8 @@
 from ..parser import validation
```

Comment:
Please expand the commit message by adding information on the purpose of your change/addition.